components/digital-agency/Partner.js

```diff
@@ -1,106 +1,95 @@
-import React, { Component } from 'react';
-import Link from 'next/link';
-import dynamic from 'next/dynamic';
-const OwlCarousel = dynamic(import('react-owl-carousel3'));
+import React, { Component, useState, useEffect } from "react";
```

Comment:
We can probably remove `Component` here too since it's changed to just a functional component

Reply:
ah yes, good point. I won't be using the import.